eclcli/bare/bareclient/ecl/common/apiclient/client.py

```diff
@@ -29,9 +29,9 @@
 import time
 
 try:
-    import simplejson as json
-except ImportError:
     import json
+except ImportError:
+    import simplejson as json
```

Comment:
Since `json` is added after Python 2.6, it is more efficient to import `json` first.